src/scripts/eastern_kingdoms/burning_steppes/blackwing_lair/boss_vaelastrasz.cpp

```diff
@@ -206,8 +206,12 @@ struct boss_vaelAI : public ScriptedAI
         if (!m_bEngaged)
         {
             m_bEngaged = true;
-            m_creature->SetRespawnDelay(43200); // 12h 43200
-            m_creature->ForcedDespawn(3600000); // 1h 3600000
+            // From 1.8: There is no longer a one-hour time restriction on the Vaelastraz the Corrupt encounter in Blackwing Lair.
+            if (sWorld.GetWowPatch() < WOW_PATCH_110)
```

Comment:
Why are you using 1.10 here, when it says 1.8 above.

Reply:
Because Grender smart. :hugging: